pyproject.toml

```diff
@@ -106,9 +106,5 @@ click = "^8.1.3"
 streamlit = ["streamlit"]
 
 
```

Comment:
still shows a change in here. please revert.

Reply:
reverted.